data/101/749/153/101749153.geojson

```diff
@@ -0,0 +1,412 @@
+    "ne:MAX_PERKM":0.0,
+    "ne:MAX_PERMI":0.0,
+    "ne:MAX_POP10":0,
+    "ne:MAX_POP20":0,
+    "ne:MAX_POP300":0,
+    "ne:MAX_POP310":0,
+    "ne:MAX_POP50":0,
+    "ne:MEAN_BBXC":0.0,
+    "ne:MEAN_BBYC":0.0,
+    "ne:MEGACITY":0,
+    "ne:MEGANAME":null,
+    "ne:MIN_AREAKM":0.0,
+    "ne:MIN_AREAMI":0.0,
+    "ne:MIN_BBXMAX":0.0,
+    "ne:MIN_BBXMIN":0.0,
+    "ne:MIN_BBYMAX":0.0,
+    "ne:MIN_BBYMIN":0.0,
+    "ne:MIN_PERKM":0.0,
+    "ne:MIN_PERMI":0.0,
+    "ne:NAME":"Vejle",
+    "ne:NAMEALT":null,
+    "ne:NAMEASCII":"Vejle",
+    "ne:NAMEDIFF":1,
+    "ne:NAMEPAR":null,
+    "ne:NATSCALE":1,
+    "ne:NOTE":null,
+    "ne:POP1950":0,
+    "ne:POP1955":0,
+    "ne:POP1960":0,
+    "ne:POP1965":0,
+    "ne:POP1970":0,
+    "ne:POP1975":0,
+    "ne:POP1980":0,
+    "ne:POP1985":0,
+    "ne:POP1990":0,
+    "ne:POP1995":0,
+    "ne:POP2000":0,
+    "ne:POP2005":0,
+    "ne:POP2010":0,
+    "ne:POP2015":0,
+    "ne:POP2020":0,
+    "ne:POP2025":0,
+    "ne:POP2050":0,
+    "ne:POP_MAX":51177,
+    "ne:POP_MIN":51177,
+    "ne:POP_OTHER":0,
+    "ne:RANK_MAX":8,
+    "ne:RANK_MIN":8,
+    "ne:SCALERANK":10,
+    "ne:SOV0NAME":"Denmark",
+    "ne:SOV_A3":"DNK",
+    "ne:TIMEZONE":"Europe/Copenhagen",
+    "ne:UN_ADM0":null,
+    "ne:UN_FID":0,
+    "ne:UN_LAT":0.0,
+    "ne:UN_LONG":0.0,
+    "ne:WORLDCITY":0,
+    "qs:a0":"Danmark",
+    "qs:a1":"*Syddanmark",
+    "qs:a1r":"0",
+    "qs:adm0":"Danmark",
+    "qs:la":"Vejle",
+    "qs:la_lc":"*DK1083630",
+    "qs:level":"locality",
+    "qs:loc":"Vejle",
+    "qs:loc_alt":"Vejle",
+    "qs:pop":0,
+    "qs:source":"KCustom EuroGlobalMap + UMZ Urban Polygons + Geonames + GeoPlanet",
+    "qs:type":"urban core",
+    "src:geom":"quattroshapes",
+    "src:geom_alt":[
+        "quattroshapes_pg"
+    ],
+    "src:lbl_centroid":"mapshaper",
+    "src:population":"geonames",
+    "wd:elevation":8.0,
+    "wd:latitude":55.708333,
+    "wd:longitude":9.533333,
+    "wd:wordcount":2799,
+    "wof:belongsto":[
+        102191581,
+        85633121,
+        1159297913,
+        85682575
+    ],
+    "wof:breaches":[],
+    "wof:concordances":{
+        "fct:id":"07c59244-8f76-11e1-848f-cfd5bf3ef515",
+        "gn:id":2610613,
+        "gp:id":558544,
+        "qs_pg:id":1091757,
+        "wd:id":"Q27116"
+    },
+    "wof:country":"DK",
+    "wof:geom_alt":[
+        "quattroshapes_pg"
+    ],
+    "wof:geomhash":"debbacd496befc18981ca3aa779641b5",
+    "wof:hierarchy":[
+        {
+            "continent_id":102191581,
+            "country_id":85633121,
+            "localadmin_id":1159297913,
+            "locality_id":101749153,
+            "region_id":85682575
+        }
+    ],
+    "wof:id":101749153,
+    "wof:lang":[
+        "dan"
+    ],
+    "wof:lastmodified":1588708678,
+    "wof:name":"Vejle",
+    "wof:parent_id":1159297913,
+    "wof:placetype":"locality",
+    "wof:population":51177,
+    "wof:population_rank":8,
+    "wof:repo":"whosonfirst-data-admin-dn",
```

Comment:
here, too

Reply:
fixed in https://github.com/whosonfirst-data/whosonfirst-data-admin-dk/pull/11/commits/c0e858a1a054baf722c1259564546539830cdfab